apps/angular/src/app/features/anime/table/table.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { AnimeService } from '@js-camp/angular/core/services/anime.service';
+import { Anime } from '@js-camp/core/models/anime';
+
+/** Table of anime. */
+@Component({
+	selector: 'camp-table',
+	templateUrl: './table.component.html',
+	styleUrls: ['./table.component.css'],
+})
+export class TableComponent implements OnInit {
+	/** Columns of table to display. */
+	public displayedColumns: string[] = ['image', 'titleEng', 'titleJpn', 'airedStart', 'type', 'status'];
```

Comment:
We prefer to encapsulate class properties as possible. So I suggest replacing `public` modifier to just `protected` instead. 
```suggestion
})
export class TableComponent implements OnInit {
	/** Columns of table to display. */
	protected readonly displayedColumns: readonly string[] = ['image', 'titleEng', 'titleJpn', 'airedStart', 'type', 'status'];
```